src/PrestaShopBundle/Resources/views/Admin/Module/Includes/tab-module-line.html.twig

```diff
@@ -0,0 +1,100 @@
+<tr>
```

Comment:
would be worth to only request attributes.

- pros: readibility
- cons: no one  ?